Core.Telemetry/IAuditLogger.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+
+using Microsoft.Cloud.InstrumentationFramework;
+
+namespace Microsoft.CloudMine.Core.Telemetry
+{
+    public interface IAuditLogger
```

Comment:
>IAuditLogger [](http://example.com/codeflow?start=21&length=12)

Oh, interesting. I thought this would stay in Core.Auditing (a separate project that depends on Telemetry), no?